### PR TITLE
Helm3 Support

### PR DESCRIPTION
**Pull request by michael-burt**
This PR adds native Helm3 support to qbec via a datasource. Chart values can be dynamically configured via the datasource provider, allowing flexibility in chart rendering based on qbec environment parameters.

This provides basic Helm support, it is by no means comprehensive when it comes to supported args. See the list of supported Helm args here: https://github.com/splunk/qbec/pull/307/files#diff-44ab577243c18af203b1825c64ccdfdac57358e759e36105647824fba1526bbcR61

### CODE REVIEW

**Comment by michael-burt** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by michael-burt** [comment]
I have read the Code of Conduct and I hereby accept the Terms

**Review by taitken-splunk — APPROVED** [review]

File: .goreleaser.yaml [hunk 1]
```diff
@@ -43,7 +44,7 @@ archives:
 
 brews:
   - name: qbec
-    tap:
+    repository:
```
Comment (michael-burt) [inline .goreleaser.yaml:47]: https://goreleaser.com/deprecations/?h=depreca#brewstap

File: .goreleaser.yaml [hunk 2]
```diff
@@ -59,7 +60,7 @@ brews:
 checksum:
   name_template: "sha256-checksums.txt"
 snapshot:
-  name_template: "{{.Version}}-next"
+  version_template: "{{.Version}}-next"
```
Comment (michael-burt) [inline .goreleaser.yaml:63]: https://goreleaser.com/deprecations/?h=depreca#snapshotname_template

File: internal/commands/eval_test.go [hunk 3]
```diff
@@ -47,13 +46,8 @@ func TestEvalWithDataSources(t *testing.T) {
 	var data map[string]interface{}
 	err = s.jsonOutput(&data)
 	require.NoError(t, err)
-
 	a := assert.New(t)
-	if runtime.GOOS == "windows" {
```
Comment (michael-burt) [inline internal/commands/eval_test.go:52]: This is no longer passing in the Windows build.

**Review by taitken-splunk — APPROVED** [review]

**Pull request merged**